population/create_glpi_computer_redfish.py

```diff
@@ -189,52 +215,63 @@ def main() -> None:
 
     urls = UrlInitialization(ip)
     Switches(switch_config)
+    error_messages = {}
+    for machine in machines:
+        print(f"Importing {machine['ipmi_ip']}")
+        try:
+            global REDFISH_BASE_URL
+            REDFISH_BASE_URL = "https://" + machine["ipmi_ip"]
+
+            REDFISH_OBJ = redfish.redfish_client(
+                base_url=REDFISH_BASE_URL,
+                username=machine["ipmi_username"],
+                password=machine["ipmi_password"],
+                default_prefix="/redfish/v1",
+                timeout=20,
+            )
+            REDFISH_OBJ.login(auth="session")
+            update_redfish_system_uri(REDFISH_OBJ, urls)
+
+            system_json = get_redfish_system(REDFISH_OBJ)
+            cpu_list = get_processor(REDFISH_OBJ)
+            ram_list = get_memory(REDFISH_OBJ)
+            storage_list = get_storage(REDFISH_OBJ)
+            nic_list, port_list = get_network(REDFISH_OBJ)
+            try:
+                REDFISH_OBJ.logout()
+            except redfish.rest.v1.RetriesExhaustedError:
```

Comment:
Do we want to log exceptions for this and/or handle other exceptions besides RetriesExhaustedError? This is a chance to consider this

Reply:
Think it's a good idea to add a logging statement. Can't think of any other exceptions that happen at this point in the script, though